guides/release/tutorial/7.md

```diff
@@ -0,0 +1,43 @@
+Actually, there is one more thing. We just put so much effort into making these pages, so we need to make sure that people can find them! The way we do that on the web is by using hyperlinks.
+
+Since Ember offers great support for URLs out-of-the-box, we _could_ just link our pages together using the `<a>` tag with the appropriate `href`. However, clicking on those links would require the browser to make a *[full-page refresh](TODO: link to full page refresh)*, which means that it would have to make a trip back to the server to fetch the page, and then load everything from scratch again.
+
+With Ember, we can do better than that! Instead of the plain-old `<a>` tag, Ember provides an alternative called `<LinkTo>`. For example, here is how you would use it on the contact page we just created:
+
+<!-- TODO: format diff -->
+```
+<div class="jumbo">
+  <div class="right tomster"></div>
+  <h2>Contact Us</h2>
+  <p>
+    Super Rentals Representatives would love to help you<br>
+    choose a destination or answer any questions you may have.
+  </p>
+  <address>
+    Super Rentals HQ
+    <p>
+      1212 Test Address Avenue<br>
+      Testington, OR 97233
+    </p>
+    <a href="tel:503.555.1212">+1 (503) 555-1212</a><br>
+    <a href="mailto:superrentalsrep@emberjs.com">superrentalsrep@emberjs.com</a>
+  </address>
+  <LinkTo @route="about" class="button">About</LinkTo>
+</div>
+```
+
+There is quite a bit going on here, so let's break it down.
+
+`<LinkTo>` is an example of a *[component](TODO: link to component)* in Ember &mdash; you can tell them apart from regular HTML tags because they start with an uppercase letter. Along with regular HTML tags, components are a key building block that we can use to build up an app's user interface.
+
+We have a lot more to say about components later, but for now, you can think of them as a way to provide *[custom tags](TODO: link to custom tags)* to supplement the built-in ones that came with the browser.
+
+The `@route=...` part is how we pass *[arguments](TODO: link to arguments)* into the component. Here, we use this to specify _which_ route we want to link to. Note that this should be the _name_ of the route, not the path, which is why we specified `"about"` instead of `"/about"`, and `"contact"` instead of `"/getting-in-touch"`.
```

Comment:
> which is why we specified `"about"` instead of `"/about"`, and `"contact"` instead of `"/getting-in-touch"`.

I don't see anywhere 'contact' was specified in a LinkTo

Reply:
fixed in my branch